readme.txt

```diff
@@ -0,0 +1,7 @@
+Take a picture of a lens against the calibration background in \examples.
```

Comment:
Are you supposed to print the calibration background in A4?

Also, can merge this into README.md?